packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -687,3 +687,9 @@ When(
   (folderName: string, count: string) =>
     fileInputUpload(parseInt(count), folderName)
 );
+
+When('I simulate network disconnection', () => {
```

Comment:
This is written as a `When`, which I think is correct, but it is written with `Then` in all scenarios

Reply:
Good question, if we define the step as `when` would you suggest using it in the test as `when`

Depending on the previous step, this may vary, example
```
When I see the first button containing "public"
Then I simulate network disconnection
```
or we could use a when after a when

Reply:
good catch, updated to `when` in all places.